src/content/guides/farmlands-pos-integration.mdoc

```diff
@@ -61,12 +59,11 @@ sequenceDiagram
 
     > The Card number is under the Cardholder’s name on the front of the card; and under the barcode on the back of the card.
 
-<img src="/farmlands-card.png" alt="The back of an example Farmlands card, displaying a barcode and a 9-digit card number" style="height: 288px;"/>
+    ![The back of an example Farmlands card, displaying a barcode and a 9-digit card number](../../assets/images/farmlands-card.png)
```

Comment:
@tom-blake Suggestion: Is there a way to set a max width? Or should we make the image itself smaller?
It's looking a little large 😅 

<img width="1107" alt="image" src="https://github.com/user-attachments/assets/7c3cfd8d-492a-47cd-805e-dc0091abcb29">